### ISSUE TITLE
USER ISSUE: cross over equator (x = 0) does strange things

### ISSUE THREAD

**Issue by Th3C0D3R** [StrangeLoopGames/EcoIssues#9546]
![20180925210408_1](https://user-images.githubusercontent.com/7798500/46036859-e625c680-c106-11e8-9a42-68f90a1a5a86.jpg)
**Version:** 0.7.7.2 beta

**Steps to Reproduce:**
take a vehicle (steam truck) and slowly drive over the equator.

you can spot the point also by looking at the smoke comming out of the engine. it will disappear shortly when you drive over the point

**Expected behavior:**
nothing happens

**Actual behavior:**
the players drop for a short time (1/2 sec) into the ground and disappears then (after I drove over the point) they appear again. also nearby vehicles (steam tractor) are launched in the air and bug away. n
**Do you have mods installed? Does issue happen when no mods are installed?:**
no

**Comment by D3nnis3n** [comment]
Glad that i did not - how i planned to do, build the capital city directly on the equator :D

**Comment by cmfrydos** [comment]
We build an entry to our central tailing storage through x=0 and have greater problems. Our truck and skid gets teleported to the top every second time,
One time it disapeared at the ground level, and when I went to the top spot I saw it falling from the sky, and at that moment the truck apeared again at the ground where my friend was. The second time it got teleported to the top level and stayed there (had some fun driving it downwards over some cliffs), and last time it got stuck a few tiles in the bottom level ceiling. One other problem: Sometimes I can only see vehicles, see and hear crafting stations when I am on the opposite site of the magical x=0 line (see image). Hard to use them that way, but a reconnect helps here. 

![20180927043711_1](https://user-images.githubusercontent.com/22503801/46120614-bd89f380-c210-11e8-9f04-07f8cfcf170e.jpg)
Notice the jump from x=0 to x=1400, and the disapearing skid (with x-values decreasing in view direction)
![20180927043714_1](https://user-images.githubusercontent.com/22503801/46120613-bd89f380-c210-11e8-9e03-964ef1dec408.jpg)
And a lonely Skid with some falling stones ;D:
![20180927032535_1](https://user-images.githubusercontent.com/22503801/46120682-f1fdaf80-c210-11e8-9da0-e91050e944aa.jpg)
We may have to think about a complicated system to get our tailings into our new storage if the teleporting thing keeps hapening..
